### DISCUSSION TITLE
Nvim font spacing smaller than in terminal

### DISCUSSION THREAD

**Discussion by Anaritus** [neovim/neovim#31056] [General]
### Problem

Using MeslLGS Nerd Font in kitty terminal
italic in font selection looks fine
![Image](https://github.com/user-attachments/assets/13bc61b6-e282-44e2-afff-5642de3c1cef)
but in neovim it does not render italic M,N symbols in full
![Image](https://github.com/user-attachments/assets/f8d985e9-59f0-48ae-b3a6-1400c84dd0ec)

### Steps to reproduce

As I said already, terminal is kitty, font is MesloLGS Nerd Font Mono

```lua
vim.api.nvim_set_hl(0, "Comment", { italic = true })
-- My comment to showcase issue MNMNMNMNM
```
Execute
nvim example.lua -u example.lua

### Expected behavior

font spacing or whatever it is should be the same as in the terminal

### Nvim version (nvim -v)

0.10.2

### Vim (not Nvim) behaves the same?

yes, 9.1

### Operating system/version

Arch Linux x86_64

### Terminal name/version

kitty

### $TERM environment variable

xterm-kitty

### Installation

AUR

**Comment by zeertzjq** [comment]
Why do you think the spacing is different? The spacing can very much be the same, and it's the characters being truncated because they don't fit in the cell width, as mentioned in <https://sw.kovidgoyal.net/kitty/faq/#some-special-symbols-are-rendered-small-truncated-in-kitty>.
